model/src/main/java/com/basistech/rosette/apimodel/EventsOptions.java

```diff
@@ -20,6 +20,10 @@
 import lombok.Builder;
 import lombok.Value;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
```

Comment:
Do you need notempty/notnull to get notblank? I know they functionally build off of each other.

Reply:
relied too much on auto-import in intellij. Fixed.